scipy/stats/_axis_nan_policy.py

```diff
@@ -0,0 +1,347 @@
+# Many scipy.stats functions support `axis` and `nan_policy` parameters.
+# When the two are combined, it can be tricky to get all the behavior just
+# right. This file contains utility functions useful for scipy.stats functions
+# that support `axis` and `nan_policy`, including a decorator that
+# automatically adds `axis` and `nan_policy` arguments to a function.
```

Comment:
We had discussed adding a `_decorators,py` file or `_util.py` file. You had suggested a more specific name than `_util.py`, and I thought that we needed more than one decorator for a `_decorators.py`. I think a good compromise is a file just for `axis` and `nan_policy` utilities, of which we have several now.